cpanfile

```diff
@@ -4,16 +4,30 @@ use warnings;    # satisfy linter
 requires 'Code::TidyAll::Plugin::SortLines::Naturally';
 requires 'Code::TidyAll::Plugin::Test::Vars';
 requires 'Code::TidyAll::Plugin::UniqueLines';
+
 requires 'Devel::Cover';
 requires 'Devel::Cover::Report::Codecov';
 requires 'Devel::Cover::Report::Coveralls';
+
+# let's also add dzil to the party using ETHER flavor :-)
+requires 'Dist::Zilla::PluginBundle::Author::ETHER';
```

Comment:
This is going to trigger a `dzil` build, which we don't need on these images.  It's also going to break on older versions of Perl.  The `Dist::Zilla` deps should be in the https://github.com/oalders/developer-perl repo.

Reply:
ok ok